src/main.js

```diff
@@ -1 +1,8 @@
-console.log("Car");
+class Car {
+  name = ''
+  position = 0
```

Comment:
```suggestion
class Car {
  name = ''
  position = 0
  
  constructor(name) {
    this.name = name;
  }
```
이렇게 constructor에서 name을 설정해주면 좋을 것 같아요!

Reply:
Step1 요구사항이 너무 간단해서 저도 생략을 많이 했네요 🫠
다음 스텝 때 추가해두겠습니다~